airflow/config_templates/default_airflow.cfg

```diff
@@ -1173,6 +1173,12 @@ job_heartbeat_sec = 5
 # how often the scheduler should run (in seconds).
 scheduler_heartbeat_sec = 5
 
+# The heartbeat frequency (in seconds) at which the scheduler will check the state of the LocalTaskJob.
+# A high value could lead to the scheduler not noticing a "zombie" LocalTaskJob quickly,
+# while a low value could mean more frequent database queries.
+# This setting is used to detect tasks that hang or tasks that have been erroneously marked as running but are not actually running.
+local_task_job_heartbeat_sec = 5
```

Comment:
We will also need to mention this config param in the https://github.com/apache/airflow/blob/main/airflow/config_templates/config.yml file so that it appears in the docs 
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html

Reply:
Thank you for pointing that out. @pankajkoti I have now added it